tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh

```diff
@@ -71,7 +71,7 @@ function install_packages() {
     iptables-persistent \
     libtcnative-1 libssl-dev libapr1-dev \
     python-flask \
-    haproxy \
+    haproxy quagga \
```

Comment:
@agneya2001 do we only need this additional package? we'll need to build new systemvm template; in that case please change the min. required and we'll need an upgrade path that does that (moves to a 4.9 systemvmtemplate). We'll also need to build/host 4.9 systemvms somewhere.